gatsby-site/src/pages/blog/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import BlogArticle from '../../components/blog-article/BlogArticle'
+import BlogMenu from '../../components/blog-menu/BlogMenu'
+import BasicPage from '../../components/BasicPage'
+import {
+  BrowserRouter,
+  Route
+} from 'react-router-dom'
+
+export default class BlogMenuPage extends React.Component {
```

Comment:
Is BlogMenu the right word for this component? Maybe it should be BlogList or something?

Reply:
I already have a blogList, but I’ll think of something. Maybe the blogList, which is just data, can be blogData, then the menu page can be blogList?